codebase/superdarn/src.lib/tk/fitacf_v3.0/src/preprocessing.c

```diff
@@ -21,6 +21,9 @@ Author(s): Keith Kotyk July 2015
 Modifications: 
     2020-03-11 Marina Schmidt (SuperDARN Canada) removed all defined 
                               constants and included rmath.h 
+    2021-05-26 Pasha Ponomarenko (SuperDARN Canada) commented out repacement of the
+                                skynoise by search noise as it caused problems with 
+                                processing BORELAIS data (no search noise)
```

Comment:
Is this a better description of the change?
```suggestion
    2021-05-26 Pasha Ponomarenko (SuperDARN Canada) check that the search noise
               is nonzero before using it to replace the skynoise when min_pwr < 1
```